lib/prependers/core_ext/acts_as_prepender.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "prependers/core_ext/acts_as_prepender"
+
+module Prependers
+  module CoreExt
+    module ActsAsPrepender
+      def acts_as_prepender(options = {})
+        if options.key?(:namespace)
+          include Prependers::Annotate::Namespace.new(options[:namespace])
+        end
+
+        if options.key?(:verify)
+          include Prependers::Annotate::Verify.new(options[:verify])
+        end
+
+        include Prependers::Prepender
```

Comment:
what about passing those options directly to `Prependers::Prepender.[]` and streamline the API a bit?

e.g.

```rb
module Acme::Animals::Dog::AddBarking
   include Prependers::Prepender[
     namespace: Acme,
     verify: 'f7175533215c39f3f3328aa5829ac6b1bb168218',
   ]

   # ...
 end
```